sdk/keyvault/Azure.Security.KeyVault.Secrets/src/KeyVaultSecret.cs

```diff
@@ -51,6 +51,7 @@ public KeyVaultSecret(string name, string value)
         /// <summary>
         /// Gets the value of the secret.
         /// </summary>
+        /// <value>This property is always null for <see cref="DeletedSecret"/>.</value>
```

Comment:
It seems like this base class doesn't offer much value (no pun intended) to the `DeletedSecret` model. Would it be more clear just to remove this as a base class?

Reply:
That would be a breaking change.

There are still a bunch of properties inherited that would have to be duplicated otherwise.